src/OpenTelemetry.Collector.Dependencies/OpenTelemetry.Collector.Dependencies.csproj

```diff
@@ -9,6 +9,10 @@
     <ProjectReference Include="..\OpenTelemetry\OpenTelemetry.csproj" />
   </ItemGroup>
 
+  <ItemGroup >
+    <PackageReference Include="System.Data.SqlClient" Version="4.7.0" />
```

Comment:
this shouldn't be needed as all reflection/PropertyFetcher is used.

Reply:
How does this work if one uses the Microsoft.Data.SqlClient package?

Reply:
Collection is done by using DiagnosticSource callbacks only. In the callbacks, use reflection to fetch required properties.
There is no dependency on either Microsoft. or System.Data package.

Reply:
Removed the dependency and replaced all calls to a reflection